proofs/cbmc/ct_get_optblocker_i32/ct_get_optblocker_i32_harness.c

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2024-2025 The mlkem-native project authors
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0 AND Apache-2.0
+
+#include "verify.h"
+
+void harness(void) { uint8_t x = mlk_ct_get_optblocker_u8(); }
```

Comment:
calling the wrong function? How can this pass?!?

Reply:
Will fix

Reply:
Changed this

Reply:
Thanks